solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -122,7 +122,7 @@ pub mod solana_ibc {
         // Before anything else, try generating a new guest block.  However, if
         // that fails it’s not an error condition.  We do this at the beginning
         // of any request.
-        ctx.accounts.chain.maybe_generate_block(&provable, Some(host_head))?;
+        // ctx.accounts.chain.maybe_generate_block(&provable, Some(host_head))?;
```

Comment:
Why is this a problem?

Reply:
While testing, we are not creating a chain, so whenever i call the deliver method, it panics because the chain is not initialized. We have to init the chain before we call the deliver method. But i need to know the config for initializing the chain though.

Reply:
I will uncomment it for now so we can merge.